protractor_tests/question.spec.js

```diff
@@ -17,13 +17,17 @@
  */
 
 var QuestionsPage = require('./questions.pageObject.js');
-var utils = require('./utils.js');
+var utils = browser.params.utils;
 
 describe('submitting questions', function() {
   it('should successfully submit wrong code to a task', function() {
     var questionsPage = new QuestionsPage();
 
+    var setting = browser.waitForAngularEnabled();
+    browser.waitForAngularEnabled(!setting);
```

Comment:
This seems a strange construct to me. Can you explain what it does?

Reply:
http://www.protractortest.org/#/api?view=ProtractorBrowser.prototype.waitForAngularEnabled